jellyfin-api/src/commonMain/kotlin-generated/org/jellyfin/sdk/api/operations/LiveTvApi.kt

```diff
@@ -1036,7 +1036,7 @@ public class LiveTvApi(
 	 *
 	 * @param includeCredentials Add the access token to the url to make an authenticated request.
 	 */
-	public fun getSchedulesDirectCountriesUrl(includeCredentials: Boolean = true): String {
+	public fun getSchedulesDirectCountriesUrl(includeCredentials: Boolean = false): String {
```

Comment:
This change is because of the new `LiveTvAccess`. We need to add this policy to the Security.kt constants or change the server (for 10.9) to also add DefaultAuthorization.